docs/source/getting_started.rst

```diff
@@ -0,0 +1,171 @@
+
+    from ankaios import Ankaios, Manifest
+
+    # Create an Ankaios object
+    ankaios = Ankaios()
+
+    # Load the manifest from the file
+    manifest = Manifest.from_file('my_manifest.yaml')
+
+    # Apply the manifest and get the result
+    ret = ankaios.apply_manifest(manifest)
+
+    # Get the workload instance name
+    wl_instance_name = ret["added_workloads"][0]
+
+    # Print the instance name
+    print(wl_instance_name)
+
+If the operation is succesfull, the result will contain a list with the added workloads that contains the workload instance name of our own.
+The workload instance name contains the name of the workload, the agent it is running on and an unique identifier.
+
+**Update a workload**
+---------------------
+
+Considering we have the above workload running, we can update certain parameters of the workload. For this example, we will update the `restartPolicy`. To be able to pinpoint
+the exact workload we want to modify, we must know the workload instance name. This can be obtained as a result when starting the workload (either using `apply_manifest` or other methods),
+deleting or modifying one. In case we don't have the workload instance name, we can take all the workloads that have the same name as the one we are looking for (or the agent).
+For simplicity, we will consider that the workload instance name is known.
+
+.. code-block:: python
+
+    from ankaios import Ankaios
+
+    # Create an Ankaios object
+    ankaios = Ankaios()
+
+    # Considering we have the workload instance name
+    wl_instance_name = WorkloadInstanceName(...)
+
+    # Get the workload base don the instance name
+    workload = ankaios.get_workload_with_instance_name(wl_instance_name)
+
+    # Update the restart policy
+    ret = workload.update_restart_policy("ALWAYS")
+
+    # Unpack the result
+    added_workloads = ret["added_workloads"]
+    deleted_workloads = ret["deleted_workloads"]
+
+Depending on the updated parameter, the workload can be restarted or not. If this is the case, the `deleted_workloads` will contain the old instance name and 
+the `added_workloads` will contain the new one.
+
+**Get the state of a workload**
+-------------------------------
+
+Having a workload running in the Ankaios system, we can retrieve the state of the workload. The state has two fields, a primary state and a substate (See `Workload States <workload_state.html>`_).
+Using the workload instance name, we can get the state of our specific workload.
+
+.. code-block:: python
+
+    from ankaios import Ankaios
+
+    # Create an Ankaios object
+    ankaios = Ankaios()
+
+    # Considering we have the workload instance name
+    wl_instance_name = WorkloadInstanceName(...)
+
+    # Get the workload state based on the instance name
+    execution_state = ankaios.get_execution_state_for_instance_name(wl_instance_name)
+
+    # Output the state
+    print(execution_state.state)
+    print(execution_state.substate)
+    print(execution_state.info)
```

Comment:
Don't forget to switch to `additional_info` here too.

Reply:
Done.